corehq/apps/reports/static/reports/javascripts/reports.download_export.js

```diff
@@ -36,4 +37,14 @@ var HQExportDownloader = function (options) {
             });
         });
     };
+    /**
+     * Attach a click event handler to the download modal "download" button that
+     * notifies google of the download event.
+     */
+    self.setUpEventTracking = function(){
```

Comment:
You might notice that this function is very similar to the identically named `setUpEventTracking` function on `ExportManager` object. This object (`HQExportDownloader`) duplicates some functionality of `ExportManager` and exists on the same page. `HQExportDownloader` should probably be done away with. Seems beyond the scope of this feature though.